packages/semantic-release-clean-package-json/README.md

```
@@ -0,0 +1,207 @@
+        "@semantic-release/release-notes-generator",
+        "@anolilab/semantic-release-pnpm",
+        "@anolilab/semantic-release-clean-package-json"
+    ]
+}
+```
+
+## Steps that are used
+
+| Step      | Description                                                                                              |
+| --------- | -------------------------------------------------------------------------------------------------------- |
+| `prepare` | Modifing the `package.json` file with the [default preserved properties](#Default-preserved-properties). |
```

Markdownlint (0.37.0)

Expected: #default-preserved-properties; Actual: #Default-preserved-properties
Link fragments should be valid

(MD051, link-fragments)